src/krkn_lib/telemetry/elastic.py

```diff
@@ -2,46 +2,46 @@
 from krkn_lib.utils.safe_logger import SafeLogger
```

Comment:
@paigerube14 these changes to elastic are simply a flake8 reformatting, there are no functional changes